CMakeLists.txt

```diff
@@ -142,9 +142,6 @@ option(
     "Build tests that compare encoding outputs to golden files. Needs AVIF_BUILD_APPS=ON and AVIF_BUILD_TESTS=ON, and depends on MP4box which can be built with ext/mp4box.sh"
     OFF
 )
-option(AVIF_ENABLE_FUZZTEST "Build avif fuzztest targets. Requires Google FuzzTest. Has no effect unless AVIF_BUILD_TESTS is ON."
-       OFF
-)
```

Comment:
I am wondering why you didn't use the `AVIF_FUZZTEST=SYSTEM/LOCAL/OFF` convention for FuzzTest.

Reply:
`SYSTEM` is most likely not available and even if it were, our dependency on fuzztest is so weak that it's better we handle it ourselves with `LOCAL` only: we know our dependency works with a particular sha1+patches. I did not try with more recent versions (as long as the CMake issues are not fixed, there is no point in trying anything newer).

Reply:
The new meaning of `AVIF_LOCAL_FUZZTEST` is subtly different from the meaning of the deprecated `AVIF_LOCAL_XXX` options. `AVIF_LOCAL_XXX` is only used when XXX is enabled, and `AVIF_LOCAL_XXX` toggles between "local XXX" and "system XXX":

```
function(set_local_or_system_option VAR DEFAULT TEXT)
    # Deal with the older way of setting options.
    if(DEFINED AVIF_LOCAL_${VAR})
        if(AVIF_LOCAL_${VAR})
            set(DEFAULT "LOCAL")
        else()
            set(DEFAULT "SYSTEM")
        endif()
        message(WARNING "Setting AVIF_LOCAL_${VAR} is deprecated. " "Set AVIF_${VAR} to ${DEFAULT} instead.")
```

After this CL, `AVIF_LOCAL_FUZZTEST` will toggle between "local FuzzTest" and "FuzzTest disabled".

It would be better if we could support `AVIF_FUZZTEST=LOCAL/OFF` instead. We can disallow setting it to `SYSTEM`. This would still be closer to the `AVIF_XXX=SYSTEM/LOCAL/OFF` convention.